local-cli/server/middleware/getDevToolsMiddleware.js

```diff
@@ -24,7 +25,42 @@ function getChromeAppName() {
   }
 }
 
-module.exports = function(options, isDebuggerConnected) {
+function launchChromeDevTools(options) {
+    var debuggerURL = 'http://localhost:' + options.port + '/debugger-ui';
+    console.log('Launching Dev Tools...');
+    opn(debuggerURL, {app: [getChromeAppName()]}, function(err) {
+        if (err) {
+            console.error('Google Chrome exited with error:', err);
+        }
+    });
+}
+
+function escapePath(path) {
+  return path.indexOf(' ') > -1
+    ? '"' + path + '"' // " Can escape paths with spaces in OS X, Windows, and *nix
+    : path;
+}
+
+function launchDevTools(options, isChromeConnected) {
+    // Explicit config always wins
+    var customDebugger = process.env.REACT_DEBUGGER;
+    if (customDebugger) {
+        var projects = options.projectRoots.map(escapePath).join(' ');
+        var command = customDebugger + ' ' + projects;
+        console.log('Starting custom debugger by executing: ' + command);
+        child_process.exec(command, function (error, stdout, stderr) {
```

Comment:
Okay, sounds good to me! Maybe clarify in the docs that the `REACT_DEBUGGER` command should exit quickly, so that the packager doesn't end up exec-ing many long-running child processes if launchDevTools is called many times.